Makefile

```diff
@@ -61,7 +78,7 @@ check-requires:
 
 install-requires:
 	@echo "*** Installing the dependencies required for testing and analysis ***"
-	dnf install -y $(TEST_DEPENDENCIES)
+	yum install -y $(TEST_DEPENDENCIES)
```

Comment:
The choice of package manager should probably be made up near the top where we decide on other variables, right?